### DISCUSSION TITLE
Logo and Navigation centered

### DISCUSSION THREAD

**Discussion by DLX23** [holger1411/understrap#2106] [Q&A] [answered]
In Understrap in the header, the logo is left-handed and the navigation right-handed.
How can I center both? Or get the navigation more left?

**Comment by IanDelMar** [comment]
Center them side by side? Which Bootstrap version? If Bootstrap 5, which navbar type?

**Comment by DLX23** [comment]
yes side by side. BS5. collapse

**Comment by IanDelMar** [comment, marked as answer]
In global-templates/navbar-collapse-bootstrap5.php line 22:
```diff
-<div class="<?php echo esc_attr( $container ); ?>">
+<div class="<?php echo esc_attr( $container ); ?> justify-content-center">
```

Add this CSS
```css
.navbar-collapse {
  flex-grow: 0;
}
```

**Reply by DLX23** [reply]
Thank you!
